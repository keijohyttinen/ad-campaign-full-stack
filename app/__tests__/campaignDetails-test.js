import 'react-native';
import React from 'react';
import CampaignDetails from '../src/components/campaignDetails';

import renderer from 'react-test-renderer';

describe('CampaignDetails test', () => {
  it('renders CampaignDetails correctly', () => {
    const props = {
      name: "Campaing Row prints TestName-2",
      navigation: {
        navigate: () => { },
        state: {
          name: "Name1",
          params: {
            id: 100000001
          },
        },
      },
    };
    const tree = renderer.create(<CampaignDetails {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
