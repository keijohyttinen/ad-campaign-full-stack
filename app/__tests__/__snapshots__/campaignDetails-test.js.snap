// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CampaignDetails test renders CampaignDetails correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
     
    Loading...
     
  </Text>
</View>
`;

exports[`CampaignDetails test renders CampaignDetails with error 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
     
    Loading...
     
  </Text>
</View>
`;
